components/formats-gpl/src/loci/formats/in/DeltavisionReader.java

```diff
@@ -161,6 +162,11 @@ public boolean isThisType(String name, boolean open) {
   /* @see loci.formats.IFormatReader#isThisType(RandomAccessInputStream) */
   @Override
   public boolean isThisType(RandomAccessInputStream stream) throws IOException {
+    // Extra sanity check; if it's a TIFF, return false immediately, to avoid
+    // false negatives.
+    if(new TiffParser(stream).isValidHeader()) {
+      return false;
+    }
```

Comment:
I think this will be redundant if the below check is fixed.  ```isValidHeader``` is checking the first four bytes; in the case of a valid header, these would be 0x49492a00 or 0x4d4d002a, which would be read as ```x``` below and should already fail the comparison with ```stream.length()```.